src/linelast_solver.cpp

```diff
@@ -417,6 +418,28 @@ void LinElastSolver::SetupDomainBCOperators()
    }
 }
 
+void LinElastSolver::SetParameterizedProblem(ParameterizedProblem *problem)
+{
+   // Set lambda and mu coeffs
+   Vector lambda(numBdr);
+   Vector mu(numBdr);
+
+   function_factory::linelast_disp::fill_lambda(lambda);
+   function_factory::linelast_disp::fill_mu(mu);
+   
+   delete lambda_c;
+   lambda_c = new PWConstCoefficient(lambda);
+
+   delete mu_c;
+   mu_c = new PWConstCoefficient(mu);
+
+   // Set bcs
+   delete init_x;
+   SetupIC(*(problem->vector_bdr_ptr[0]));
+   AddBCFunction(*(problem->vector_bdr_ptr[0]), 1);
```

Comment:
As mentioned above, `SetParameterizedProblem` should be problem-agnostic. We should not assume/limit our problems to always have boundary attribute `1` and `2`. This is in fact why we have `vector_bdr_ptr` as `Array`, so that boundary conditions can be set in a problem-agnostic way.

See `PoissonSolver::SetParameterizedProblem` and `StokesSolver::SetParameterizedProblem` as examples of how these functions are used in a problem-agnostic way.